admin/services/pin-generation.service.js

```diff
@@ -50,7 +50,7 @@ pinGenerationService.getPupils = async (schoolId, sortField, sortDirection) => {
  * @returns {Boolean}
  */
 pinGenerationService.isValid = async (p) => {
-  const checkCount = await checkDataService.count({ pupilId: p._id, checkStartedAt: { $ne: null } })
+  const checkCount = await checkDataService.sqlNumberOfChecksStartedByPupil(p._id)
```

Comment:
I feel there should be a `get` or `fetch` in there somewhere.  Or a `find`... so many to choose from.

Reply:
👍